README.md

```diff
@@ -118,6 +118,22 @@ To stop use:
 make inventory-down
 ```
 
+#### Kessel Inventory + Kessel Relations using built binaries
+
+Inventory and Relations can also be run locally using built binaries, but the default config for Inventory will conflict with Relations.
+
+To run Relations locally, see the [Running Locally](https://github.com/project-kessel/inventory-api/tree/main?tab=readme-ov-file#running-locally) section of the Relations README
+
```

Comment:
```suggestion
Instructions to run Spicedb and Relations API

Run Spicedb containers 

`make spicedb-up`

Run Relations API locally
`make run`

```

Reply:
Going to link to the section in their docs of spinning up spicedb in case they ever change it.

Reply:
Updated with links to reduce re-writing existing docs and to hopefully get updates if they are changed over there as well

Reply:
The main challenges are with importing schema and determining which schema to use. We can add some additional notes with zed command example